src/ui/Avatar/Avatar.tsx

```diff
@@ -140,13 +147,26 @@ export const Avatar = ({
 }) => {
   const avatarPath = getAvatarPath(path);
 
+  const { data: coSoul } = useQuery(
+    [QUERY_KEY_PROFILE_TOTAL_PGIVE, address],
+    () => queryProfilePgive(address),
```

Comment:
hmm, do we make a ton of queries for this now? prob fine but might be nice to list as a tech debt improvement to try and batch queries

Reply:
@CryptoGraffe what do you think about this?

<img width="492" alt="Screenshot 2023-06-26 at 11 22 19 AM" src="https://github.com/coordinape/coordinape/assets/83605543/70798aa2-72c3-458a-abb8-1cd290dfea7a">

Reply:
Can we rather join in the cosoul data whenever we fetch the avatar info instead of doing these extra queries?